internal/release/RELEASE-NOTES

```diff
@@ -2,89 +2,4 @@
 
 ## NEW
 
-- **Support for item states**: You can now fetch an item's state using the SDK. `ItemOverview` exposes one of two states: `Active` or `Archived`.
-    - `Active`: An item located inside a vault. (Default)
-    - `Archived`: An item that has been moved to the Archive. 1Password doesn't include archived items in search results or suggest them when you fill in apps and browsers. You can keep archived items as long as you'd like.
-- **Filtering listed items by state**: You can now filter the results of the item list function by item state.
-
-## FIXED
-
-- **Deleting Archived Items:** The SDK now supports deleting items from the archive.
-
-## ⚠️ BREAKING CHANGES ⚠️
-This release contains breaking changes for two functions in the Go SDK.
-
-**Vault listing**
-
-* The function name has changed from `ListAll` to `List`. To use this in your code, replace:
-```go
-vaults, err := client.Vaults().ListAll(context.Background(), vaultID)
-// error handling
-```
-with:
-```go
-vaults, err := client.Vaults().List(context.Background(), vaultID)
-// error handling
-```
-
-* The return type of the vault listing function has changed from `*Iterator[VaultOverview]` to `[]VaultOverview`. To use this in your code, replace:
-
-```go
-for {
-	vault, err := vaults.Next()
-	if errors.Is(err, onepassword.ErrorIteratorDone) {
-		break
-	} else if err != nil {
-		// error handling
-	}
-	// using the vault overview
-}
-```
-with:
-```go
-for _, vault := range vaults {
-	// using the vault overview
-}
-```
-**Item listing**
-
-* The function name has changed from `ListAll` to `List`. To use this in your code, replace:
-```go
-overviews, err := client.Items().ListAll(context.Background())
-// error handling
-```
-with:
-```go
-overviews, err := client.Items().List(
-	context.Background(),
-	vaultID,
-	onepassword.NewItemListFilterTypeVariantByState(
-		&onepassword.ItemListFilterByStateInner{
-			Active:   true,
-			Archived: true,
-		},
-	),
-)
-// error handling
-```
-
-* The return type of the item listing function has changed from `*Iterator[ItemOverview]` to `[]ItemOverview`. To use this in your code, replace:
-```go
-for {
-	overview, err := overviews.Next()
-	if errors.Is(err, onepassword.ErrorIteratorDone) {
-		break
-	} else if err != nil {
-		// error handling
-	}
-	// using the item overview
-}
-```
-with:
-```go
-for _, overview := range overviews {
-	// using the item overview
-}
-```
-
-This does not affect any code that's already deployed, and will not take effect in your codebase until you choose to update to version 0.3.0 or later of the 1Password Go SDK.
+- `VaultOverview` has `created_at` and `updated_at` fields
```

Comment:
Let's double check that there weren't any other features added to the SDK since we last released